cdap-api/src/main/java/co/cask/cdap/api/workflow/WorkflowToken.java

```diff
@@ -89,9 +171,18 @@
   Map<String, Map<String, Long>> getMapReduceCounters();
 
   /**
-   * Return true if the {@link WorkflowToken} contains the specified key.
+   * Return true if the {@link WorkflowToken} contains the specified key
+   * for a {@link Scope#USER} scope.
    * @param key the key to be tested for the presence in the {@link WorkflowToken}
    * @return the result of the test
    */
   boolean containsKey(String key);
+
+  /**
+   * Return true if the {@link WorkflowToken} contains the specified key.
+   * @param key the key to be tested for the presence in the {@link WorkflowToken}
+   * @param scope the {@link WorkflowToken.Scope} for the key
+   * @return the result of the test
+   */
+  boolean containsKey(String key, Scope scope);
```

Comment:
Removed these methods and now not allowing null key or value in the WorkflowToken.